product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -126,15 +128,15 @@ To enable you to view valid values to use in the configuration file for BigAnima
 ```shell
 biganimal cluster show-architectures
 __OUTPUT__
-┌───────────────────────────────────────────────┐
-│ Architecture                                  │
-├────────┬───────────────────────────┬──────────┤
-│ ID     │ Name                      │ Status   │
-├────────┼───────────────────────────┼──────────┤
-│ ha     │ High Availability         │ enabled  │
-│ pgd    │ Extreme High Availability │ disabled │
-│ single │ Single Node               │ enabled  │
-└────────┴───────────────────────────┴──────────┘
+┌───────────────────────────────────────────────────────────────┐
+│ Architecture                                                  │
+├────────┬───────────────────────────────────────────┬──────────┤
+│ ID     │ Name                                      │ Status   │
+├────────┼───────────────────────────────────────────┼──────────┤
+│ ha     │ Primary/Standby High Availability         │ enabled  │
```

Comment:
I don't believe the name change was implemented in 3.0. See https://enterprisedb.atlassian.net/browse/UPM-23196.

Reply:
Ah, I see. I did not know that. Thanks! Will fix.